test/e2e/persistentips_test.go

```diff
@@ -40,7 +40,23 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var _ = Describe("Persistent IPs", func() {
+const (
+	secondaryLogicalNetworkInterfaceName = "multus"
+	nadName                              = "l2-net-attach-def"
+)
+
+const (
+	rolePrimary   = "primary"
+	roleSecondary = "secondary"
+)
+
+type testParams struct {
+	role    string
+	ipsFrom func(vmi *kubevirtv1.VirtualMachineInstance) ([]string, error)
```

Comment:
This should be the expectaionts so let's rename this to `expectedIPsFrom`

Reply:
It's not the expectedIPs. In all cases, it's the "actualIPs", as you can see here:
```
Expect(testenv.ThisVMI(vmi)()).Should(testenv.MatchIPs(ipsFrom, ConsistOf(vmiIPsBeforeRestart)))
```

the expectation is that the "actualIPs" to be wither empty/consist of the same ips before migration/brave/...

Reply:
right, all good.